assets/js/blocks/classic-template/constants.ts

```diff
@@ -14,10 +14,16 @@ export const TYPES = {
 	productCatalog: 'product-catalog',
 	productTaxonomy: 'product-taxonomy',
 	productSearchResults: 'product-search-results',
+	orderConfirmation: 'order-confirmation',
+	cart: 'cart',
+	checkout: 'checkout',
+	checkoutHeader: 'checkout-header',
 };
 export const PLACEHOLDERS = {
 	singleProduct: 'single-product',
 	archiveProduct: 'archive-product',
+	orderConfirmation: 'fallback',
```

Comment:
There should be no conflict here because `fallback` is defined within `conversionConfig`—its not specifically for the `orderConfirmation`. If another placeholder was added which also did not need its own special placeholder, it too would use `fallback`.

Ref:
```
const blockifiedFallbackConfig = {
	isConversionPossible: () => false,
	getBlockifiedTemplate: () => [],
	getDescription: () => '',
	onClickCallback: () => void 0,
};

const conversionConfig: { [ key: string ]: BlockifiedTemplateConfig } = {
	...
	fallback: blockifiedFallbackConfig,
};
```